### DISCUSSION TITLE
Provide instructions for running with debug logs

### DISCUSSION THREAD

**Discussion by tmgstevens** [sauljabin/kaskade#42] [Q&A] [answered]
I'm seeing an issue where Kaskade doesn't render because of `Failed to get committed offsets - timed out`
All credentials are correct.
Please provide instructions (or mechanism) to run with debug logs so that we can work out what it's trying to do and failing

**Comment by sauljabin** [comment, marked as answer]
Hi @tmgstevens thanks for asking.

Here you have some guidelines https://github.com/sauljabin/kaskade/blob/main/DEVELOPMENT.md.

But in general you have to:
```bash
poetry install
poetry shell
textual console --port 7342
```

In another terminal:
```bash
poetry shell
textual run --port 7342 --dev -c kaskade admin -b localhost:19092
```

Also kaskade has a default logger you can use:
```bash
tail -f ~/.kaskade/kaskade.log
```

Then in the code you could:
```python
from kaskade import logger
...
logger.error("my error")
...
```
